src/Http/ProxyRouter.php

```diff
@@ -259,6 +259,9 @@ public function isTargetAPhpScript(): bool
      */
     public function proxify(): void
     {
+        if ($this->handleWellKnownURIs()) {
+            return;
+        }
```

Comment:
This is a good idea, but it should not be done in the `proxify()` method. It would be better to have a new `handleRedirects()` public method called as soon as possible after the router instanciation. Also, this method should probably have a `string $uri_prefix` parameter, to be compatible with a GLPI served by an `Alias` directive.

Reply:
Is the prefix relevant when it only checks the end if the path contains `.well-known` and only uses the part of the path after that, and uses relative paths for redirects?